packages/terra-date-time-picker/package.json

```diff
@@ -0,0 +1,68 @@
+{
+  "name": "terra-date-time-picker",
+  "main": "lib/DateTimePicker.js",
+  "version": "0.0.0",
+  "description": "The DateTimePicker component has a date picker for selecting date and a time input for entering time",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/cerner/terra-core.git"
+  },
+  "keywords": [
+    "Cerner",
+    "Terra",
+    "terra-date-time-picker",
+    "DateTimePicker",
+    "UI"
+  ],
+  "author": "Cerner Corporation",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/cerner/terra-core/issues"
+  },
+  "homepage": "https://github.com/cerner/terra-core#readme",
+  "devDependencies": {
```

Comment:
`terra-app-delegate`, `terra-date-picker` and `terra-modal-manger` can be dropped from devDependencies

Reply:
I'm guessing these are included to facilitate with tests scenarios around including date-picker in a modal. I'm fine keeping them as devDependecies for this case.

Reply:
Yes, these are included to facilitate testing.